src/components/InternalOverlay/readme.md

```diff
@@ -5,6 +5,8 @@ import { ButtonIcon } from 'react-rainbow-components';
 import { useRef, useState } from 'react';
 import styled from 'styled-components';
 import RenderIf from '../RenderIf';
+import useOutsideClick from '../../libs/hooks/useOutsideClick';
+import useWindowResize from '../../libs/hooks/useWindowResize';
```

Comment:
I think we should start documenting every internal implementation as part of the internal session, I will avoid code duplication since the library is huge and something we can end up repeating same implementation.

things under internal session will only for internal use and we specific to consumers of the library that the API of internal could change and it won't be considered breaking change